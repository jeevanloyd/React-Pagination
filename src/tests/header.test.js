import React from 'react';
import { shallow } from 'enzyme';
import HeaderComponent from '../components/HeaderComponent';

test('should test Header component', () => {
 const wrapper = shallow(<HeaderComponent />);
 expect(wrapper).toMatchSnapshot();
});