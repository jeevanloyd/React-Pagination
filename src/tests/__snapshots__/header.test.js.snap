// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Header component 1`] = `
<div
  className="d-flex bg-dark text-white justify-content-center py-4"
>
  All Products
</div>
`;
